<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerParametersControlWidget</class>
 <widget class="QWidget" name="ScannerParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScannerParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>algo</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="algoComboBox">
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Brightness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hue</string>
        </property>
       </item>
       <property name="toolTip">
         <string>algo</string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>red threshold</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="redThresholdSpinBox">
       <property name="toolTip">
         <string>red threshold</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>height</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="heightSpinBox">
       <property name="toolTip">
         <string>height</string>
       </property>
     </widget>
    </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>algoComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScannerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>redThresholdSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScannerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>heightSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScannerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
